src/ArticleType.php

```diff
@@ -24,6 +24,8 @@ abstract class ArticleType
 
     public const GameHash = 10;
 
+    public const SetClaim = 11;
```

Comment:
This will conflict with #1019. I'll update that to 12, but there will likely still be a merge conflict resolution required.

Reply:
Alright, thanks.